lib/github_service/commands/cross_repo_test.rb

```diff
@@ -207,6 +220,37 @@ def valid?
         true
       end
 
+      def validate_repo_names
+        invalid_repo_names = (@test_repos + @repos).select { |r| r.include?(" ") }.sort.uniq
+        if invalid_repo_names.any?
+          message = "@#{issuer} 'cross-repo-test(s)' was given invalid repo names and cannot continue\n\n"
+          invalid_repo_names.each do |repo|
+            message << "* `#{repo}`\n"
+          end
+
+          issue.add_comment(message)
+          return false
+        end
+
+        true
```

Comment:
Minor, but the flow could be a little cleaner if we return true early like
```
invalid_repo_names = (@test_repos + @repos).select { |r| r.include?(" ") }.sort.uniq
return true if invalid_repo_names.empty?

message = "@#{issuer} 'cr...
...

false
```

Reply:
Oh yeah I like that